app/renderer/js/pages/preference/add-certificate.ts

```diff
@@ -1,20 +1,28 @@
 'use-strict';
 
-const { dialog } = require('electron').remote;
-const path = require('path');
+import { remote, OpenDialogOptions } from 'electron';
```

Comment:
Not sure if electron has this api `OpenDialogOptions` ?

Reply:
Made another comment in this file.

Reply:
Yes it not a api, the `OpenDialogOptions` is a type that typescript can import.

Reply:
Got it.